pkg/fakerp/fakerp.go

```diff
@@ -201,6 +203,15 @@ func createOrUpdateWrapper(ctx context.Context, p api.Plugin, log *logrus.Entry,
 		return nil, err
 	}
 
+	if !isUpdate {
+		wID, wKey, err := createWorkspace(ctx, os.Getenv("AZURE_SUBSCRIPTION_ID"), cs.Location, os.Getenv("RESOURCEGROUP"))
```

Comment:
Note the wID here is not the UUID that we need, it looks like this "/subscriptions/<sub>/resourcegroups/angus-dev/providers/microsoft.operationalinsights/workspaces/openshift".